docs/src/routes/docs/[...4]wallets/[...12]gnosis/+page.md

```diff
@@ -29,7 +29,7 @@ npm install @web3-onboard/gnosis @safe-global/safe-apps-provider @safe-global/sa
 
 ```typescript
 type GnosisOptions = {
-  whitelistedDomains: RegExp[]
```

Comment:
Can we keep the naming here and change how its passed within the module instead?